app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def five_recent_comments
+    comments.limit(3)
```

Comment:
- [ ] @Yosefgeda great work adding a method for returning 5 recent comments, however your method current returns 3 comments and doenst guarantee they will be most recent, kindly check on my suggest on the `three_recent_posts` method on user model and apply the same here.